app/containers/Forms/ProductionFields.tsx

```diff
@@ -61,11 +62,15 @@ export const ProductionFieldsComponent: React.FunctionComponent<Props> = ({
     const productUnits = query.allProducts.edges.find(
       ({node}) => node.rowId === productRowId
     )?.node.units;
+    const requiresEmissionAllocation = query.allProducts.edges.find(
+      ({node}) => node.rowId === productRowId
+    )?.node.requiresEmissionAllocation;
     onChange({
       ...formData,
       productRowId,
       productUnits,
-      additionalData: {productUnits}
+      requiresEmissionAllocation,
+      additionalData: {productUnits, requiresEmissionAllocation}
```

Comment:
Additional data will be removed following the latest requirement. This can stay here for now, I'll remove it in the next PR